backend/forms/graphql/limited_survey.py

```diff
@@ -0,0 +1,32 @@
+import graphene
+from django.conf import settings
+from graphene_django import DjangoObjectType
+
+from ..models.survey import Survey
+
+DEFAULT_LANGUAGE: str = settings.LANGUAGE_CODE
+
+
+class LimitedSurveyType(DjangoObjectType):
+    title = graphene.Field(graphene.String, lang=graphene.String())
+
+    @staticmethod
+    def resolve_title(parent: Survey, info, lang: str = DEFAULT_LANGUAGE):
```

Comment:
```suggestion
    def resolve_title(parent: Survey, info, lang: str = DEFAULT_LANGUAGE) -> str | None:
```

Reply:
It's my outspoken and Kompassi's written policy that return types are optional on functions. However, in this instance, it's unintuitive but correct that the title can be null: it comes from the language versions, and there is a valid state in which the survey does not yet have any language versions (when the survey is created but no language versions have yet been added) so I shall acquiesce.